ToolBox/AlphaVantageDownloader/AlphaVantageDataDownloader.cs

```diff
@@ -0,0 +1,168 @@
+﻿using CsvHelper;
```

Comment:
Please add the LEAN Apache2.0 header to all files -- (can see on other code files)